docroot/themes/contrib/civictheme/civictheme_library/components/00-base/menu/menu.twig

```diff
@@ -46,8 +46,9 @@
       ] %}
 
       {% set item_attributes = item.attributes|default('') %}
-      {% set aria_expanded = item.in_active_trail and item.is_expanded ? 'aria-expanded="true"' : 'aria-expanded="false"' %}
-      {% if is_collapsible %}
+      {% set aria_expanded = '' %}
+      {% if item.below and is_collapsible %}
```

Comment:
an item can be collapsible only if it has children - otherwise there is nothing to collapse :)